doc/src/learn/cryptography/sui-offline-signing.md

```diff
@@ -4,16 +4,15 @@ title: Offline Signing
 
 Sui supports offline signing, which is signing transactions using a device not connected to a Sui network, or in a wallet implemented in a different programming language without relying on the Sui key store. The steps to implement offline signing include:
 
- 1. Serialize the data for signing.
- 1. Sign the serialized data. Put the serialized data in a location to sign (such as the wallet of your choice, or tools in other programming languages) and to produce a signature with the corresponding public key.
- 1. Execute the signed transaction.
+1. Serialize the data for signing.
+2. Sign the serialized data. Put the serialized data in a location to sign (such as the wallet of your choice, or tools in other programming languages) and to produce a signature with the corresponding public key.
```

Comment:
Iirc there is a reason that we always put  - 1. (Rendering  automatically adds the counter) @randall-Mysten to confirm